_data/devops-checklist.yml

```diff
@@ -0,0 +1,690 @@
+
+- category: Networking
+  items:
+    - title: Set up VPCs
+      description: |
+        Don't use the Default VPC, as everything in it is publicly accessible by default. Instead, create one or more
+        custom <a href="https://aws.amazon.com/vpc/" target="_blank">Virtual Private Clouds (VPC)</a>, each with their
+        own IP address range (see
+        <a href="https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Subnets.html#VPC_Sizing" target="_blank">VPC and subnet sizing</a>),
+        and deploy all of your apps into those VPCs.
+
+    - title: Set up subnets
+      description: |
+        Create three "tiers" of
+        <a href="https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Subnets.html" target="_blank">subnets</a>
+        in each VPC: public, private-app, private-persistence. The public subnets are directly accessible from the
+        public Internet and should only be used for a small number of highly locked down, user-facing services, such as
+        load balancers and Bastion Hosts. The private-apps subnets are only accessible from within the VPC from the
+        public subnets and should be used to run your apps (Auto Scaling Groups, Docker containers, etc.). The
+        private-persistence subnets are also only accessible from within the VPC from the private-app subnets (but NOT
+        the public subnets) and should be used to run all your data stores (RDS, ElastiCache, etc.). See
+        <a href="https://www.whaletech.co/2014/10/02/reference-vpc-architecture.html" target="_blank">A Reference VPC Architecture</a>.
+
+    - title: Configure Network ACLs
+      description: |
+        Create <a href="https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_ACLs.html" target="_blank">Network Access Control Lists (NACLs)</a>
+        to control what traffic can go between different subnets. We recommend allowing the public subnets to receive
+        traffic from anywhere, the private-app subnets to only receive traffic from the public subnets, and the
+        private-persistence subnets to only receive traffic from the private-app subnets.
+
+    - title: Configure Security Groups
+      description: |
+        Every AWS resource (e.g., EC2 Instances, Load Balancers, RDS DBs, etc.) has a
+        <a href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-network-security.html" target="_blank">Security Group</a>
+        that acts as a firewall, controlling what traffic is allowed in and out of that resource. By default, no
+        traffic is allowed in or out. Follow the
+        <a href="https://en.wikipedia.org/wiki/Principle_of_least_privilege" target="_blank">Principle of Least Privilege</a>
+        and open up the absolute minimum number of ports you can for each resource. When opening up a port, you can
+        also specify either the CIDR block (IP address range) or ID of another Security Group that are allowed to
```

Comment:
`or ID of another Security Group that are allowed to` -> ...that _is_ allowed to...

Reply:
Fixed! https://github.com/gruntwork-io/gruntwork-io.github.io/pull/45/commits/ac969266fa45512affe5434f70111b488f8c4eb8